src/Microsoft.Data.Analysis/GroupBy.cs

```diff
@@ -70,7 +72,7 @@ public abstract class GroupBy
         public abstract DataFrame Mean(params string[] columnNames);
     }
 
-    public class GroupBy<TKey> : GroupBy
+    public class GroupBy<TKey> : GroupBy 
```

Comment:
(nit) this whitespace change should be reverted

Reply:
fixed